jobs/jumpbox/templates/bin/watcher

```diff
@@ -202,9 +202,9 @@ EOF
   fi
 <% end %>
 
-  sshd_configure PasswordAuthentication <%= p('jumpbox.ssh.password_auth') == "true" ? "yes" : "no" %>
-  sshd_configure AllowAgentForwarding <%= p('jumpbox.ssh.allow_agent_forwarding') == "true" ? "yes" : "no" -%>
-  sshd_configure AllowTcpForwarding <%= p('jumpbox.ssh.allow_tcp_forwarding') == "true" ? "yes" : "no" -%>
+  sshd_configure PasswordAuthentication <%= p('jumpbox.ssh.password_auth') == true ? "yes" : "no" %>
+  sshd_configure AllowAgentForwarding <%= p('jumpbox.ssh.allow_agent_forwarding') == true ? "yes" : "no" %>
+  sshd_configure AllowTcpForwarding <%= p('jumpbox.ssh.allow_tcp_forwarding') == true ? "yes" : "no" %>
```

Comment:
Are we sure this is a boolean true and not the string "true"?  If so, when did it change, and also you should never test `== true`, you should just use the value as the conditional.